### PR TITLE
Removing "OpenSeadragon Cropping Tool" as website is gone

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -104,7 +104,6 @@ Various tools for working with images such as cropping tools.
 
 - [Leaflet-IIIF Cropping](https://bl.ocks.org/mejackreed/6936585f435b60aa9451ae2bc1c199f2) - Example of using Leaflet to provide IIIF cropping.
 - [Stanford Cropper](https://github.com/lizfischer/iiif-tools#cropper) - Simple image cropper.
-- [OpenSeadragon Cropping Tool](http://sul-dlss.github.io/iiif-cropper/demo/) - Script to allow for cropping an image from within OpenSeadragon.
```
Comment (mejackreed) [inline readme.md:107]: The url is now here: http://sul-dlss-deprecated.github.io/iiif-cropper/demo/

I would have thought GitHub would have redirected that automatically :(

Comment (spekulatius) [reply readme.md:107]: Ah, okay. I've updated the link instead

Comment (spekulatius) [reply readme.md:107]: Could you confirm this is okay @mejackreed ?

**Comment by regisrob** [comment]
👍

**Comment by spekulatius** [comment]
Thanks for merging it in @regisrob

**Pull request merged**